frontend/src/services/user.ts

```diff
@@ -0,0 +1,25 @@
+import {AuthCredentials} from "../models/authCredentials";
+import {ILoggedUser} from "../types/loggedUser";
+import {UserEntity} from "../models/userEntity";
+import {UserType} from "../types/UserType";
+
+export const logInService = async (authCredentials: AuthCredentials): Promise<string> => {
+   return "eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJjdXN0b21lcjFAZ21haWwuY29tIiwiaWF0IjoxNzExOTYyMDM4fQ.g5oDNUFe6MwsfP7syojOKMTd75OXiSL0VGCPEc2aoHg";
+}
+
+export const getUser = async (token: string): Promise<ILoggedUser> => {
```

Comment:
You need to send token with all requests except sign in. Come up with a solution to add it to the requests automatically instead of manual approach as it is now